hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -747,15 +747,15 @@
       clock_srcs: {clk_i: "main"},
       clock_group: "infra",
       reset_connections: {rst_ni: "lc"},
-      base_addrs: {rom: "0x00008000", regs: "0x411e0000"},
+      base_addrs: {rom: "0x00010000", regs: "0x411e0000"},
       memory: {
         rom: {
           label:              "rom",
           swaccess:           "ro",
           data_intg_passthru: "true",
           exec:               "True",
           byte_write:         "False",
-          size:               "0x8000",
+          size:               "0x10000",
```

Comment:
This and the edits above in this file are the only changes in this PR - the rest is all autogenerated!